runtime/gc_base/RootScanner.cpp

```diff
@@ -356,6 +356,20 @@ MM_RootScanner::doClassSlot(J9Class *classPtr)
 	/* ignore class slots by default */
 }
 
+#if JAVA_SPEC_VERSION >= 24
+/**
+ * @todo Provide function documentation
+ */
+void
+MM_RootScanner::doContinuationSlot(J9Object **slotPtr, GC_ContinuationSlotIterator *continuationSlotIterator)
+{
+	/* ensure that this isn't a slot pointing into the gap (only matters for split heap VMs) */
+	if (!_extensions->heap->objectIsInGap(*slotPtr)) {
```

Comment:
I understand this is what doStackSlot is doing too, but I think this check is unnecessary in base RootScanner.
It's specific to standard collectors (and only some configurations there)
Again, I don't want to risk, and let's leave it.